packages/commonwealth/client/scripts/hooks/useAnimation.ts

```diff
@@ -0,0 +1,25 @@
+import { useEffect, useState } from 'react';
+
+export const useAnimation = () => {
+  const [animationStyles, setAnimationStyles] = useState({
+    opacity: 0,
+    transform: 'translateY(40px)',
```

Comment:
shouldn't this be dynamic?

Reply:
it might be yeah, but because I am using it only in one place now and I am not sure if we will use it anywhere else, I just hardcoded it. My proposition is to leave it as it is and adjust when we would need more flexibility.

Reply:
ah cool, resolving.